### ISSUE TITLE
SBB false positive

### ISSUE THREAD

**Issue by igorgiovannini** [martinschaible/rspamd-rules#80]
Hi Martin,

First, thanks a lot for your great work, these rules are a great starting point, very well maintained! 😍

I just did some checks after a couple of days since the installation, and noticed a false positive about an SBB e-mail that hurts customers in Switzerland.

```
[Envelope From] From
[bounce_sbb@order.info.sbb.ch] sbbclient@order.info.sbb.ch

Subject
Your online purchase from SBB
```

**Symbols**

```
sender.address.orgbrandprod (12) [sbbclient@order.info.sbb.ch]
BAYES_HAM (-3) [100.00%]
RWL_MAILSPIKE_GOOD (-1) [93.191.164.64:from]
DCC_BULK (0.9) [fuz2=many,rep=45%]
DMARC_POLICY_ALLOW (-0.5) [order.info.sbb.ch,reject]
MX_INVALID (0.5) []
FORGED_SENDER (0.3) [sbbclient@order.info.sbb.ch,bounce_sbb@order.info.sbb.ch]
R_SPF_ALLOW (-0.2) [+ip4:93.191.164.0/24]
R_DKIM_ALLOW (-0.2) [inxserver.com:s=202504,order.info.sbb.ch:s=inx58838273]
MIME_GOOD (-0.1) [multipart/related,multipart/alternative,text/plain]
ONCE_RECEIVED (0.1)
MANY_INVISIBLE_PARTS (0.05) [1]
```

Actually the mail gets a score of `8.85`, and according to my Rspamd configuration, it goes through the `Add header` action.
The mail is not blocked, but will get moved to Spam folder.

I might potentially mitigate it, but I think that the `sender.address.orgbrandprod` rule should be slightly modified to prevent this false positive detection.

What do you think?

Thanks a lot!

Cheers 👋🏻

**Comment by igorgiovannini** [comment]
Hi Martin,

Thanks a lot. Actually SBB is not the only company in Switzerland that uses `Inxmail` as relay server... Swiss Post uses it as well.

I only have some doubts about this specific IP whitelisting... I just went back on my e-mails and I found for instance a postshop newsletter delivered via `93.191.164.147`, while the previous one has been delivered via `93.191.164.64` (see: the same IP as the one used by SBB yesterday), then the previous one before via `93.191.162.82`, ... (I didn't continue checking then)

The whole `93.191.164.0/24` subnet is owned by Inxmail, same for the `93.191.162.0/24` subnet.

Then the risk by whitelisting those two subnets would be that some malicious/spammy senders are abusing of some addresses within those subnets, meh... 🙃

What do you think? I saw that you already whitelisted some IPs within the same subnet for `DPD Deutschland GmbH`...
